### PR TITLE
Fixed the Navbar height and user login page

### PR DESCRIPTION
**Pull request by arpittyagirocks**
# 💡 [REQUEST] - <UI Improvement> Navbar and User login page #192

## Description of Changes

- I have fixed the height of navbar which was causing the navbar to appear thick and also it was covering the content
- I have changed the positioning of the main form in user login page 
- In the Orignal screenshot 2 you can clearly see how bad it looks

## Checklist:

- [x] My code adheres to the established style guidelines of this project.
- [x] I have conducted a self-review of my code.
- [x] I have included comments in areas that may be difficult to understand.
- [x] I have made corresponding updates to the project documentation.
- [x] My changes have not introduced any new warnings.

## Screenshots

|           Original           |           Updated           |
| :--------------------------: | :-------------------------: |
|  | link |
|  | link |

## Feedback is appreciated

### CODE REVIEW

**Comment by arpittyagirocks** [comment]
### Please provide any update about merging this pull request

File: public/sitemap-0.xml [hunk 1]
```diff
@@ -1,17 +1,17 @@
...
-<url><loc>https://grabtern.com/mentorRegister</loc><lastmod>2023-05-25T15:20:22.186Z</lastmod><changefreq>daily</changefreq><priority>0.7</priority></url>
-<url><loc>https://grabtern.com/mentorRegisterSendCV</loc><lastmod>2023-05-25T15:20:22.186Z</lastmod><changefreq>daily</changefreq><priority>0.7</priority></url>
-<url><loc>https://grabtern.com/mentors</loc><lastmod>2023-05-25T15:20:22.186Z</lastmod><changefreq>daily</changefreq><priority>0.7</priority></url>
-<url><loc>https://grabtern.com/privacy</loc><lastmod>2023-05-25T15:20:22.186Z</lastmod><changefreq>daily</changefreq><priority>0.7</priority></url>
-<url><loc>https://grabtern.com/refundandcancellation</loc><lastmod>2023-05-25T15:20:22.186Z</lastmod><changefreq>daily</changefreq><priority>0.7</priority></url>
-<url><loc>https://grabtern.com/register</loc><lastmod>2023-05-25T15:20:22.186Z</lastmod><changefreq>daily</changefreq><priority>0.7</priority></url>
-<url><loc>https://grabtern.com/termsandcondition</loc><lastmod>2023-05-25T15:20:22.186Z</lastmod><changefreq>daily</changefreq><priority>0.7</priority></url>
```
Comment (anmode) [inline public/sitemap-0.xml:16]: I think it is good to do in this pR. put this sitemap-0.xml in gitignore. please

**Review by anmode — APPROVED** [review]
Look cool! A couple of changes-
1. Maintain the UI formity of login, registration and forget password container
2. Can you remove the scroll bar? ( That is coming in registration and login.
3. UI needs to be improved in mobile view

**Comment by arpittyagirocks** [comment]
## Did the changes you wanted
- [ ] Maintain the UI formity of login, registration and forget password container
- [ ] Can you remove the scroll bar? ( That is coming in registration and login.
- [ ] UI needs to be improved in mobile view
- [ ] Added sitemap-0.xml in gitignore
### I hope you like it @anmode

**Comment by anmode** [comment]
Great! But UI formity did not achieved?
see
Screenshot 2023-05-31 at 8 33 18 PM
Screenshot 2023-05-31 at 8 33 29 PM
Screenshot 2023-05-31 at 8 33 23 PM

**Comment by arpittyagirocks** [comment]
### Added UI formiity just as you said. I hope i came over your expectations,
### please merge the changes @anmode

**Comment by anmode** [comment]
I'll mark it as level 3. If you can include Nav and mentor card fixing too. It is your wish otherwise Level 2 is good. and I'll assign it.

File: styles/globals.css [hunk 2]
```diff
@@ -1223,28 +1222,7 @@ textarea {
...
-  background-color: #0275d8;
-  color: #fff;
-  font-size: 18px;
-  font-weight: 500;
-  border: none;
-  border-radius: 5px;
-  padding: 15px;
-  width: 100%;
-  cursor: pointer;
-}
```
[the remark this answers is not in the document]
Comment (arpittyagirocks) [reply styles/globals.css:1247]: initially forgot password page was using these designs in it.
I added same designs and classes to the forgot password page as login page and registration page, so we don't need these
I hope i am clear about it

**Comment by arpittyagirocks** [comment]
### Thank you, but i am okay with level 2

**Comment by anmode** [comment]
Congrats!

**Comment by arpittyagirocks** [comment]
Thank you

**Pull request merged**